### PR TITLE
CNV-41522: Fix customize IT

### PR DESCRIPTION
**Pull request by metalice**
## 📝 Description

Fix the flow of customize IT
 

Edit disk
Edit nic
Cancel redirect
Adding SSH
Boot managment

## 🎥 Demo

> Please add a video or an image of the behavior/changes

### CODE REVIEW

File: src/views/catalog/CustomizeInstanceType/components/CustomizeITVMFooter.tsx [hunk 1]
```diff
@@ -28,6 +28,7 @@ import './CustomizeITVMFooter.scss';
 const CustomizeITVMFooter: FC = () => {
   const { t } = useKubevirtTranslation();
   const navigate = useNavigate();
+  const [activeNamespace] = useActiveNamespace();
```
Comment (upalatucci) [inline src/views/catalog/CustomizeInstanceType/components/CustomizeITVMFooter.tsx:31]: Can this page be rendered on 'all-namespaces' ? In this case activeNamespace will be `ALL_NAMESPACES_SESSION_KEY`

File: src/utils/store/customizeInstanceType.ts [hunk 2]
```diff
@@ -37,12 +37,14 @@ export const updateCustomizeInstanceType: UpdateCustomizeInstanceType = (
   updateValues,
 ): V1VirtualMachine => {
   let vm = vmSignal.value;
+
```
Comment (pcbailey) [inline src/utils/store/customizeInstanceType.ts:40]: 🎉

File: src/views/catalog/wizard/tabs/network/components/list/NetworkInterfaceActions.tsx [hunk 3]
```diff
@@ -8,23 +8,27 @@ import TabModal from '@kubevirt-utils/components/TabModal/TabModal';
...
   nicName,
   nicPresentation,
+  onUpdateVM,
 }) => {
   const { t } = useKubevirtTranslation();
-  const { updateVM, vm } = useWizardVMContext();
+  const { updateVM, vm: vmContext } = useWizardVMContext();
   const { createModal } = useModal();
-
+  const vm = vmSignal.value || vmContext;
```
Comment (pcbailey) [inline src/views/catalog/wizard/tabs/network/components/list/NetworkInterfaceActions.tsx:30]: Is this a temporary solution or will we always use context as a fallback? If it's not temporary, what's the value in having two store solutions?

Comment (metalice) [reply src/views/catalog/wizard/tabs/network/components/list/NetworkInterfaceActions.tsx:30]: in the end i would like to remove all use of context and stay only with signals so its temp until a wider refactor

File: src/views/catalog/CustomizeInstanceType/components/CustomizeITVMFooter.tsx (hunk 1 above)
Comment (metalice) [reply src/views/catalog/CustomizeInstanceType/components/CustomizeITVMFooter.tsx:31]: should be fine, as this is only for the cancel button, but a wider check and adjustment will be followed so changes will be made again

**Pull request merged**